src/main/webapp/script.js

```diff
@@ -110,16 +105,14 @@ function loadHeatmap(data) {
             radius: 100, // arbitrary value used to make the points easily visible
         });
     }
-    aqLayer.setData([]);
+    //aqLayer.setData([]);
```

Comment:
Keep or keep not, there is no commented out code.

Reply:
Oops, I meant to delete but I forgot that I had only commented out to test if it would break things

Reply:
Was that a Yoda reference?

Reply:
A Yoda reference, indeed it was.